packages/fx-core/resource/package.nls.json

```diff
@@ -614,6 +614,17 @@
   "core.collaboration.error.dotEnvFileNotExist": "Failed to find your .env File.",
   "core.collaboration.error.failedToLoadDotEnvFile": "Failed to load your .env File. Reason: %s",
   "core.collaboration.error.failedToGetTeamsAppId": "Failed to find Teams app id in your .env file. Please correctly set the value with key '%s' in your .env file.",
+  "core.collaboration.error.manifestFileNotExist": "Failed to find your manifest file: %s.",
```

Comment:
please use this Error:  FileNotFoundError

Reply:
Updated, thx!